searx/templates/simple/elements/search_url.html

```diff
@@ -1,6 +1,7 @@
 <div id="search_url" role="complementary" aria-labelledby="search_url-title">
   <details class="sidebar-collapsable">
     <summary class="title" id="search_url-title">{{ _('Search URL') }}</summary>
+    <button id="copy_url" type="submit">{{ _('Copy') }}</button>
```

Comment:
If you change for
`<button id="copy_url" type="submit" data-copied-label="{{ _('Copied') }}">{{ _('Copy') }}</button>`

then `searxng.on('#copy_url', 'click', function () {` can use `data-copied-label`  and it can be translated.

(`data-copied-label` or whatever attribute name)

Reply:
thanks, i've improved it.